.github/workflows/build-linux-amd64.yml

```diff
@@ -37,18 +37,31 @@ jobs:
           export GOARCH=${GOARCH}
           export GOOS=${GOOS}
           make build-linux-with-shared-library
-          cd ./build
+          pushd build
           mkdir -p initia_${VERSION}
           mv libmovevm.so initia_${VERSION}/libmovevm.${ARCH_NAME}.so
           mv libcompiler.so initia_${VERSION}/libcompiler.${ARCH_NAME}.so
           mv initiad initia_${VERSION}/
           tar -czvf initia_${VERSION}_Linux_${ARCH_NAME}.tar.gz initia_${VERSION}
+          popd
+
+      - name: Static Build for Linux AMD64
+        run: |
+          export GOARCH=${GOARCH}
+          export GOOS=${GOOS}
+          make build-linux
+          pushd build
+          mkdir -p initia_${VERSION}
```

Comment:
need to use different Directory from the shared library one

Reply:
thanks for noticing me!